src/test/disemvowelTest.js

```diff
@@ -0,0 +1,21 @@
+import chai, { expect } from 'chai'
+import disemvowel from '../main/disemvowel'
+
+
+  describe('disemvowel', () => {
+
+    it('does the thing', () => {
+    })
+
+    describe('disemvowel', () => {
+      it('removes the vowels from a given string', () => {
+
+        expect(disemvowel('yo check this out'))
+        .to.equal('ychckthst')
+        expect(disemvowel('yaldcakhcvasdnskcbjahvLZKxavkaacauceyficksgukUFugsjnnnxnnxFSHDBACKUgeuagfaukfavheywftcgjvcepmjgacd khufd'))
+        .to.equal('yldckhcvsdnskcbjhvLZKxvkccyfcksgkFgsjnnnxnnxFSHDBCKggfkfvhywftcgjvcpmjgcdkhfd')
+        expect(disemvowel('yo    check this out'))
+        .to.equal('ychckthst')
```

Comment:
each of these tests should have their own `it('what it is expectuing'...)` For example the second seems to be testing for your functions ability to handle a really long input so you should say that i.e. `it('Removes all vowels regardless of string size', ...)`